app/models/project.js

```diff
@@ -31,8 +41,80 @@ export default class ProjectModel extends Model {
 
   @hasMany('milestone', { async: false }) milestones;
 
+  // Attributes for dashboard view
+  @attr() tab;
+
+  @attr() teammemberrole;
+
   @attr() applicantteam;
 
+  @computed('milestones')
+  get publicReviewPlannedStartDate() {
+    const cbMilestone = this.milestones.find(milestone => milestone.milestone === COMMUNITY_BOARD_REFERRAL_MILESTONE_ID);
+    if (cbMilestone) {
+      return cbMilestone.plannedstartdate;
+    }
+    return null;
+  }
```

Comment:
Does this computed return a string? What happens when you pass this through moment.js or something that expects a string, and it's null?

Reply:
I'm designing it so that it's up to the template or controller to check whether the computed returns a string or is null. Thinking that returning a null value is useful for indicating when publicReviewPlannedStartDate (or any of the other date fields) is not applicable.

Reply:
Although returning `null` is more appropriate for the `displayMilestoneActual<Start/End>Date` fields. Theoretically there should always be a community board review planned start date?

Reply:
seems `{{moment-format null}}` yields "Invalid date"